### ISSUE TITLE
[feature] Display replies?

### ISSUE THREAD

**Issue by Ezwen** [moko256/twitlatte#30]
Hi! Loving twitlatte so far :).

I was wondering whether you would consider displaying all answers given to a tweet, when a tap is made on a tweet?

**Comment by sbadux** [comment]
This is an importan missing feature IMHO. I hope you'll consider to add it. Anyway, really thanks for your work :)

**Comment by JonasT** [comment]
Yea, it's the main reason I am only occasionally using it since I use threads a lot, and without this I'm just seeing distinct tweets. Would love to see this added! Also, please add a way of not just seeing the replies, but also jumping to any *previous* replies (they don't need to be shown, but there should be a button to jump back by one, so there's at least *some* way to go back) - unless there's already one I missed!

**Comment by ckuethe** [comment]
Would very much like to see replies and threads.

**Comment by zoj613** [comment]
yeah I was wondering why it doesnt show replies at all. can you please add this feature, thanks.

**Comment by creativetrendsapps** [comment]
bump.

Was coming here to add this as an "issue" as well.

**Comment by geofrey** [comment]
+1 this.

I'm currently trying out more clients to see whether it's just Twidere that can't load replies or if Twitter have broken 3rd-party apps on purpose again. Not even having the capability to read threads is a defect.

**Comment by yaomtc** [comment]
With Mastodon I can see the number of replies, but I can't see the replies. With Twitter I can't see either.

**Comment by RayLennox** [comment]
Bump. Agree with OP.

Is this a feature, to not see replies? Or is it a bug?

Any modern client fetches replies, so it would be nice to have it in Twitlatte.
